src/backend/commands/player/toxic/kick_from_vehicle.cpp

```diff
@@ -15,15 +15,36 @@ namespace big
 
 		virtual void execute(player_ptr player, const command_arguments& _args, const std::shared_ptr<command_context> ctx) override
 		{
-			auto vehicle = player->get_current_vehicle();
+			int vehicle = PED::GET_VEHICLE_PED_IS_IN(PLAYER::GET_PLAYER_PED(player->id()), false);
 
-			if (!vehicle || !vehicle->m_net_object)
+			if (!vehicle)
```

Comment:
This check was there to make sure people didn't execute this in SP. Why did you remove it? You also replaced the vehicle type from pointer to a native type. 0 does not indicate that the object was not found (-1 does), but now that you've changed the fundamental type, you now need to use ENTITY::DOES_ENTITY_EXIST, as the ! check was checking for nullptr.